### DISCUSSION TITLE
Move a child endpoint back to the top level

### DISCUSSION THREAD

**Discussion by koraybilgi** [mockapi-io/docs#2] [Q&A] [answered]
Hi, 
I accidentally made an endpoint a child. And I was not able to undo this action. 
Is there a way to move a child endpoint back to the top level?

Thanks.

**Comment by nsmirn0v** [comment, marked as answer]
Unfortunately, there is no way to do this at the moment. However, I'm planning to add this functionality in the future. Hope this helps!
